src/app/tests/suites/certification/ci-pics-values

```diff
@@ -440,4 +439,63 @@ MCORE.ACL.Administrator=1
 OTAS.S.M.DelayedActionTime=1
 OTAS.S.M.UserConsentNeeded=1
 OTAC.C.M.AnnounceOTAProvider=1
-OTAC.C.M.NotifyUpdateApplied=1
\ No newline at end of file
+OTAC.C.M.NotifyUpdateApplied=1
+
+#On/Off Cluster
+OO.S.A0000=1
+OO.S.A4000=1
+OO.S.A4001=1
+OO.S.A4002=1
+OO.S.A4003=1
+OO.S.C00.Rsp=1
+OO.S.C01.Rsp=1
+OO.S.C02.Rsp=1
+OO.S.C40.Rsp=1
+OO.S.C41.Rsp=1
+OO.S.C42.Rsp=1
+OO.M.ManuallyControlled=1
+OO.C.C00.Tx=1
+OO.C.C01.Tx=1
+OO.C.C02.Tx=1
+OO.C.C40.Tx=1
+OO.C.C41.Tx=1
+OO.C.C42.Tx=1
+
+#Actions Cluster
+ACT.S.A0000=1
+ACT.S.A0001=1
+ACT.S.A0002=1
+ACT.S.M.FillActionList=1
+ACT.S.M.FillEndpointLists=1
+ACT.C.C00.Tx=0
+ACT.C.C01.Tx=0
+ACT.C.C02.Tx=0
+ACT.C.C03.Tx=0
+ACT.C.C04.Tx=0
+ACT.C.C05.Tx=0
+ACT.C.C06.Tx=0
+ACT.C.C07.Tx=0
+ACT.C.C08.Tx=0
+ACT.C.C09.Tx=0
+ACT.C.C0a.Tx=0
+ACT.C.C0b.Tx=0
```

Comment:
Why are these 0?  CI can in fact send those commands, I am sure.

Reply:
@bzbarsky-apple i have set these attributes values as '0' based on this verification [sheet](https://docs.google.com/spreadsheets/d/1h4d4J90KAxFqUL_cXZHyM3pHjHCEXzFITkQhN2KMqDY/edit#gid=1782756114).in this sheet testplan team mentioned these attributes are not implemented in Sdk yet,please refer this screenshot 
![2022-07-04_14-05](https://user-images.githubusercontent.com/57476670/177116400-dbe68632-c1c0-4ce9-9a02-c44029bb3bc4.png)